docs/static/troubleshooting.asciidoc

```diff
@@ -59,8 +59,8 @@ Running Logstash with Java 11 results in warnings similar to these:
 [source,sh]
 -----
 WARNING: An illegal reflective access operation has occurred
-WARNING: Illegal reflective access by org.jruby.util.SecurityHelper (file:/Users/chrisuser/logstash-6.7.0/logstash-core/lib/jars/jruby-complete-9.2.6.0.jar) to field java.lang.reflect.Field.modifiers
-WARNING: Please consider reporting this to the maintainers of org.jruby.util.SecurityHelper
+WARNING: Illegal reflective access by org.jruby.ext.openssl.SecurityHelper (file:/var/folders/cg/p_vcxtdj32d4ssw5cz3n43th0000gn/T/jruby-44535/jruby16549305496655381936jopenssl.jar) to field java.security.MessageDigest.provider
```

Comment:
Can we redact some of the bits that we expect to be environment/build-specific?

```suggestion
WARNING: Illegal reflective access by org.jruby.ext.openssl.SecurityHelper (file:/{...}/jruby{...}jopenssl.jar) to field java.security.MessageDigest.provider
```

Reply:
sure! I was happy to remove the explicit mention of a user's home directory, but I agree no one wants to see how quirky someone else's tmpdir is.

Reply:
done 👍